erpnext/selling/setup_wizard_slide/add_a_few_customers/add_a_few_customers.json

```diff
@@ -7,14 +7,14 @@
  "domains": [],
  "help_links": [
   {
-   "label": "Customers",
+   "label": "Learn More",
```

Comment:
Change the help link label for Suppliers slide too. Make it `Learn More` from `Suppliers`

Reply:
+1